packages/zoe/src/contracts/autoswap.js

```diff
@@ -1,8 +1,10 @@
 // @ts-check
+// @jessie-check
 
 import { Far } from '@endo/marshal';
 import { assert } from '@agoric/assert';
 import { AmountMath, isNatValue } from '@agoric/ertp';
+import { makeMap } from 'jessie.js';
```

Comment:
@michaelfig is that the right way to import from the Jessie std library? No `@something/something` ?

Reply:
Yes, absolutely!  `'jessie.js'` is the stdlib package.